_posts/2016-11-17-JAX-London-2016.md

```diff
@@ -0,0 +1,691 @@
+---
+layout: post
+authors: [yannick_de_turck]
+title: 'JAX London 2016'
+image: /img/jax-london-2016/jax-london-logo.png
+tags: [Domain-Driven Design, DDD, Software Craftmanship, Microservices]
+category: JAX London
+comments: true
+---
+> JAX London is a four-day conference for cutting-edge software engineers and enterprise-level professionals. JAX brings together the world’s leading innovators in the fields of Java, microservices, Continuous Delivery and DevOps.
+> For this year's slogan they decided on: "Create, Innovate, Code".
+> Ordina was present at JAX London on the 11th and 12th of October 2016 where one of our colleagues, Bart Blommaerts, also presented a talk on The Serverless Cloud.
+> In this blogpost we want to give the highlights of some of the talks be followed.
+
+----------
+
+# Day 1
+
+## Introduction to JAX London - Sebastian Meyen
+
+<img class="float-image p-image" src="{{ '/img/jax-london-2016/sebastian-meyen.jpeg' | prepend: site.baseurl }}" alt="Sebastian Meyen" style="width: 100px"/>
+
+To start off, [Sebastian Meyen](https://twitter.com/smeyen), program chair of the JAX conferences, gave an introduction to JAX London.
+He mentioned that only half of the attendants of JAX originate from the UK, making it an international conference.
+JAX is all about "openess", we should be celebrating open source and embrace open source thinking as it makes our code smarter.
+Java is one of the most powerful ecosystems out there according to him and it comes with a unique culture.
+
+Sebastian explained that there are different cultures within a company:
+
+* Pioneers: They go into the wild and experiment and although they might fail often, they're looking at what the next thing might be
```

Comment:
what the next thing => what the next big thing 

?? not sure, but the next things sounds odd